Feature/infrastructure vpc subnets

adrianz-kainos opened the pull request.

adrianz-kainos committed:
Add VPC subnets, dhcp, routing and internet gateway

adrianz-kainos committed:
Add security rules for SSH, HTTP, HTTPS, DNS, PostgresSQL and Microservices

adrianz-kainos committed:
Reformat code

adrianz-kainos committed:
Add separate security_groups

Refactor code

xjedam commented on Infrastructure/RDS/variables.tf:
```diff
@@ -2,8 +2,11 @@ variable "DB_USERNAME" {
+  type = "list"
 }
+//variable "security_group_id" {
+//}
```
this should be removed if its not used

adrianz-kainos replied:
Done, thx.

adrianz-kainos committed:
Merge branch 'master' into feature/infrastructure-vpc-subnets

adrianz-kainos committed:
Add an API Gateway stage with a client certificate

adrianz-kainos committed:
Add GenerateCertificate.sh and place it in ./TerraformInfrastructure.sh
Rename region to AWS_REGION and move it to an environment variable
Update README.md

adrianz-kainos committed:
Terraform var bugfix. Refactor code. Expand GenerateCertificate.sh.

adrianz-kainos committed:
Add AWS Lambda module to create a proxy between the API Gateway and microservices

adrianz-kainos committed:
(WIP) Add Lambda proxy function between the API Gateway and the microservices
